src/gluonts/transform/feature.py

```diff
@@ -555,3 +555,39 @@ def map_transform(self, data: DataEntry, is_train: bool) -> DataEntry:
         )
 
         return data
+
+
+class AddTrailingZeros(SimpleTransformation):
```

Comment:
How about `CountTrailingZeros`? "Add" makes it sound like some right-padding is done on the target

Reply:
Yes, thank you!